standalone-metastore/pom.xml

```
@@ -78,7 +78,7 @@
     <junit.version>4.11</junit.version>
     <libfb303.version>0.9.3</libfb303.version>
     <libthrift.version>0.9.3</libthrift.version>
-    <log4j2.version>2.8.2</log4j2.version>
+    <log4j2.version>2.17.0</log4j2.version>
```

```suggestion
    <log4j2.version>2.17.1</log4j2.version>
```
To cover CVE-2021-44832